### PR TITLE
Croatian Army Woodland loadout

### PR DESCRIPTION
**Pull request by TheLustyDragon**
Added loadout.
Doesn't contain UAV operator.

### CODE REVIEW

**Comment by beaar** [comment]
Every role should be covered by the loadout. You need to include the UAV operator (and if you removed any other roles, them too) loadout even if you can't find appropriate equipment for it.

You also need an entry in the CfgLoadouts.hpp file that links the loadout. Look at how the other loadouts are linked and do the same for yours.
https://github.com/Sniperhid/1tac_misc/blob/master/addons/tm_tmf_loadouts/CfgLoadouts.hpp

**Comment by TheLustyDragon** [comment]
Properly uploaded loadout to loadouts folder.
Updated CfgLoadouts.hpp to include the new loadout.
Updated loadout to mention me as author.
Loadout now also contains the UAV operator, no other classes were removed.

I think I did it right this time.

**Comment by beaar** [comment]
Waiting for the second loadout

**Comment by TheLustyDragon** [comment]
Added the 1992 loadout entry to CfgLoadouts.hpp and added tooltips to both loadouts.

**Comment by beaar** [comment]
There's a lot of issues with the 1992 loadout. Vest and uniforms that can't fit magazines etc. You should test the loadouts more thoroughly. Run TMF Autotest in the editor.

File: addons/tm_tmf_loadouts/loadouts/cro_army_1992.hpp [hunk 1]
```diff
@@ -58,13 +58,14 @@ class r : baseMan
...
     displayName = "Grenadier";
+	backpack[] = {"rhs_sidor"};
     magazines[] +=
     {
         LIST_5("rhs_mag_rgd5"),
         LIST_3("rhs_mag_rdg2_white")
     };
 };
-class car : r
+class car
```
Comment (beaar) [inline addons/tm_tmf_loadouts/loadouts/cro_army_1992.hpp]: Why did you remove the parent class?

**Comment by beaar** [comment]
The "armed suits" loadouts are both missing a CfgLoadouts.hpp entry. They probably belong in the 'Other' category.

**Comment by beaar** [comment]
I want you to put the suits loadouts in the 'Other' category since they don't really have any national identity.

File: addons/tm_tmf_loadouts/loadouts/cro_army_1992.hpp [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+        "ItemMap",
+        "ItemCompass",
+        "ItemRadio",
+        "ItemWatch",
+        "Binocular",
+    };
+	backpack[] = {"rhs_rpg_empty"};
+    backpackItems[] =
+    {
+        LIST_2("rhs_mag_9k38_rocket ")
```
Comment (beaar) [inline addons/tm_tmf_loadouts/loadouts/cro_army_1992.hpp]: trailing space

**Comment by beaar** [comment]
You need to test the loadouts more thoroughly yourself, so we don't have to go back and forth like this, wasting both my and your time. There are a lot of autotest errors and at least one popup error because of the trailing space I highlighted in the earlier comment.
